tests/e2e/multi_svc_util.go

```diff
@@ -0,0 +1,285 @@
+func isAlarmPresentOnDatacenter(datacenter string, alarmToVerify string, alarmShouldExists bool) (bool, error) {
+	alarmCmd := govcLoginCmd() + "govc events /" + datacenter + " | grep 'warning'"
+	framework.Logf("Get alarms from datacenter - command : %s", alarmCmd)
+	waitErr := wait.PollImmediate(healthStatusPollInterval, pollTimeoutSixMin, func() (bool, error) {
+		result, err := exec.Command("/bin/sh", "-c", alarmCmd).Output()
+		if err != nil {
+			// handling ExitError here which occurs sometimes
+			if exitErr, ok := err.(*exec.ExitError); ok {
+				// Get the stderr output
+				stderr := string(exitErr.Stderr)
+				framework.Logf("Stderr:", stderr)
+				return false, nil
+			} else {
+				framework.Logf("Not an ExitError:", err)
+				return false, fmt.Errorf("error fetching alarms details : %v", err)
+			}
+
+		}
+		if string(result) != "" {
+			alarms := strings.Split(string(result), "\n")
+			for _, alarm := range alarms {
+				alarm = strings.TrimSpace(alarm)
+				// Checking if required alarm appears
+				if strings.Contains(alarm, alarmToVerify) {
+					if alarmShouldExists {
+						framework.Logf("Required alarm Found : %s", alarm)
+						return true, nil
+					} else {
+						// In case alarmShouldExists is false, but alarm is found
+						return false, nil
+					}
+				}
+			}
+			// after above iteration, in case alarm not present and param alarmShouldExists also false
+			if !alarmShouldExists {
+				framework.Logf("Required alarm not found")
+				return true, nil
+			}
+		}
+		return false, nil
+	})
+	return true, waitErr
+}
+
+// removeEsxiHostFromCluster method is used to remove esxi hosts from cluster
+func removeEsxiHostFromCluster(datacenter string, cluster string, hostIP string) (bool, error) {
+	removeHostFromCluster := govcLoginCmd() + "govc object.mv /" + datacenter + "/host/" + cluster +
+		"/" + hostIP + " /" + datacenter + "/host/"
+	framework.Logf("Remove an ESXi host from cluster command : %s", removeHostFromCluster)
+	_, err := exec.Command("/bin/sh", "-c", removeHostFromCluster).Output()
+	if err != nil {
+		framework.Logf("Error: %v\n", err)
+		return false, fmt.Errorf("couldn't execute command: %s, error: %s",
+			removeHostFromCluster, err)
+	}
+	return true, nil
+}
+
+// moveHostToCluster method is used to move a host to cluster
+func moveHostToCluster(clusterPath string, hostIP string) error {
+	moveHostToCluster := govcLoginCmd() + "govc cluster.mv -cluster " + clusterPath + " " + hostIP
+	framework.Logf("Move a host to cluster command : %s", moveHostToCluster)
+	_, err := exec.Command("/bin/sh", "-c", moveHostToCluster).Output()
+	if err != nil {
+		framework.Logf("Error: %v\n", err)
+		return fmt.Errorf("couldn't execute command: %s, error: %s",
+			moveHostToCluster, err)
+	}
+	return nil
+}
+
+// getVcSessionIDforSupervisor method returns list of vc session id for a supervisor id and returns error if any
+func getVcSessionIDforSupervisor(supervisorId string) ([]string, error) {
```

Comment:
change `getVcSessionIDforSupervisor` to `getVcSessionIDsforSupervisor`
also try to limit this to CSI sessions

Reply:
Done